utils.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const got = require('got');
 const childProcess = require('child_process');
 
-const REQUESTED_VERSION = process.env.CHROMIUM_VERSION || '86.0.0';
+const REQUESTED_VERSION = process.env.CHROMIUM_VERSION || '121.0.6156.3';
```

Comment:
latest as of time of PR.  technically, only the major version (aka milestone) is needed ... didnt want to change the variable name tho as the goal was to keep this PR as light as possible.